bin/tachyon

```diff
@@ -116,6 +117,59 @@ function runTest {
   return $RET
 }
 
+function journalCrashTest {
+  if [ "$#" -lt 1 ]; then
+    echo "Usage: tachyon journalCrashTest <test-duration-in-seconds> [-options]"
```

Comment:
I'll use HelpFormatter.printHelp to print out the usage and only put it in the java file.
Then, I'll kill/restart Master for multiple times in the java file instead of a specified time duration.